docs/data/charts/styling/styling.md

```diff
@@ -102,6 +102,21 @@ This configuration can be used in Bar Charts to set colors according to string c
 }
 ```
 
+## Overlay
+
+Charts have a _loading_ and _noData_ overlays that appears if
+
+- `loading` prop is set to `true`.
+- There is no data to display.
+
+{{"demo": "Overlay.js"}}
```

Comment:
This demo looks good, but I find it difficult to understand when I open the code. Maybe divide it into two different sections?

```suggestion
Charts have a _loading_ and _noData_ overlays that appears if

- `loading` prop is set to `true`.
- There is no data to display.

{{"demo": "Overlay.js"}}

You can also provide the axes data in order to display them while loading the data. Once the data is loaded, you can update them to show the correct ranges, if not know beforehand.

// second part of demo above
```

Reply:
I removed this sentence because it was a bit confusing

> Once the data is loaded, you can update them to show the correct ranges, if not know beforehand.

I also added H3 titles mostly to get space around the demo, to get a more structured page and simplify the reading

Reply:
No worries, mine was just an example 👍